src/components/ExternalLinkBlock/index.tsx

```diff
@@ -31,6 +33,10 @@ const ExternalLinkBlock = ({
 }: ExternalLinkBlockProps) => {
   const settings = useSettings();
   const { locale } = useLocale();
+  const isMovie = mediaType === MediaType.MOVIE;
+  const isTv = mediaType === MediaType.TV;
+  const isPerson = mediaType === 'person';
+  const imdbType = isPerson ? 'name' : 'title';
```

Comment:
Not really a big fan of creating dedicated variables to store very simple checks used only once.